release-notes.md

```diff
@@ -1,11 +1,11 @@
 # Tucana-20.7.0
+### Deployment scripts ###
+    - php deployment/updates/scripts/2024_03_05_deploy_event_notification_batch_job_email_notification_failure.php
```

Comment:
@TalDubovKaltura  why did you bump the "deployment scripts" section up?

Reply:
@hilak This section came from the merge I did